src/duckstation-qt/gamelistwidget.cpp

```diff
@@ -582,12 +582,18 @@ QVariant GameListModel::data(const QModelIndex& index, int role, const GameList:
           else
             return {};
         }
-
-        default:
-          return {};
       }
     }
 
+    case Qt::TextAlignmentRole:
+    {
+      const int column = index.column();
+      if (column == Column_FileSize || column == Column_UncompressedSize)
+        return (Qt::AlignRight | Qt::AlignVCenter).toInt();
```

Comment:
`toInt()` is necessary because there is no implicit conversion from `QFlags<whatever>` to QVariant.